### DISCUSSION TITLE
Lock file maintenance schedule needs separate override

### DISCUSSION THREAD

**Discussion by matthewnessworthy** [renovatebot/renovate#16262] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

Renovate creates PRs outside of the defined schedule

It should run at `after 9:00am and before 1:00pm every weekday`
But instead runs at `before 5am on monday`

### Relevant debug logs

<details><summary>Relavant config</summary>

```
{
    "extends": [
        "config:base"
    ],
    "dependencyDashboard": true,
    "schedule": [
        "after 9:00am and before 1:00pm every weekday"
    ],
    "timezone": "Africa/Johannesburg",
    "major": {
        "enabled": true
    },
    "minor": {
        "enabled": false
    },
    "patch": {
        "enabled": false
    },
    "lockFileMaintenance": {
        "enabled": true
    }
}
```

</details>

<details><summary>Logs</summary>

```
DEBUG: Checking schedule(before 5am on monday, Africa/Johannesburg)(branch="renovate/lock-file-maintenance")
DEBUG: Found timezone(branch="renovate/lock-file-maintenance")
{
  "timezone": "Africa/Johannesburg"
}
DEBUG: Adjusting now for timezone(branch="renovate/lock-file-maintenance")
DEBUG: Checking 1 schedule(s)(branch="renovate/lock-file-maintenance")
DEBUG: Checking schedule "before 5am on monday"(branch="renovate/lock-file-maintenance")
{
  "parsedSchedule": {
    "schedules": [
      {
        "t_b": [
          18000
        ],
        "d": [
          2
        ]
      }
    ],
    "exceptions": [],
    "error": -1
  }
}
DEBUG: Matches schedule before 5am on monday(branch="renovate/lock-file-maintenance")
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment, marked as answer]
`lockFileMaintenance` has its own default `schedule`: https://docs.renovatebot.com/configuration-options/#lockfilemaintenance

To override that, you need to do like this:

```
{
  "lockFileMaintenance": {
    "schedule": [
        "after 9:00am and before 1:00pm every weekday"
    ]
  }
}
```

**Reply by rarkins** [reply]
But I recommend you don't do it every day, you will probably find that annoying because you'll probably get lock file maintenance PRs on most days.

**Reply by matthewnessworthy** [reply]
thanks for clarifying

i'm currently running it every day on some select repos to work out any bugs and complications that would occur for the rest of the org

**Reply by rarkins** [reply]
I mean specifically lock file maintenance. It defaults to weekly schedule because such PRs are usually not required non-stop or even daily. For everything else, a daily schedule is usually fine
